web/cobrands/northamptonshire/assets.js

```diff
@@ -231,29 +205,6 @@ fixmystreet.assets.add(northants_road_defaults, {
     ]
 });
 
-var barrier_style = new OpenLayers.Style({
```

Comment:
Why are we removing the barrier and the footpath/bridleway/etc styling? For the latter, is that data not available in the new layer? But for the former, it's the whole layer, looks like?

Reply:
The barriers are now spot assets, so this style is no longer relevant as it's designed for line assets.

As for the footpath/bridleway/etc styling, the data isn't available in the new layer, yep.

Reply:
Oh and the rest of the barrier layer has been moved into the array of layers at the top of the file, so it gets added as part of the loop.;